code/game/machinery/cloning.dm

```diff
@@ -264,7 +264,7 @@
 		clonemind.transfer_to(H)
 	else if(!(!experimental && body_only))
 		current_insurance = insurance
-		offer_to_ghost(H)
+		offer_to_ghost(H, account_id)
```

Comment:
there should be a system in place for specific memories copying (even if it is just account IDs for now)
some shouldn't copy like objectives but there is a whole memories system in code used by changelings.

Reply:
It is due to experimental clones, not you

Reply:
well, I am going to remove this. it's a bit bloating